Q2)The United Way raises money for community charity activities. Recently, in one community, the
fundraising committee was concerned about whether there is a difference in the proportion of
employees who give to United Way depending on whether the employer is a private business or a
government agency. A random sample of people who had been contacted about contributing last year
was selected. Of those contacted, 70 worked for a private business and 50 worked for a government
agency. For the 70 private-sector employees, the mean contribution was RM230.25 with a standard
deviation equal to RM55.52. For the 50 government employees in the sample the mean and standard
deviation were RM309.45 and RM61.75, respectively.

a) Based on these sample data and alpha = 0.05, what should be concluded? Be sure to show the
decision rule.

We are interested in determining whether there is a difference in the proportion of employees who give to United Way whether the employee is a private business or government agency

Ho : u1 = u2, Ha: u1 not equal to u2

In [1]:
import pandas as pd
import numpy as np
import scipy.stats as st 
import math as m

In [2]:
n1 = 70 # private sector employees
n2 = 50 #government sector employees
x1 = 230.25 #sample mean for private sector employees
x2 = 309.25 #sample mean for government sector employees
sd1 = 55.52 #standard deviation for private sector employees
sd2 = 61.75 #standard deviation for government sector employees  

In [6]:
#two tailed test
#get critical value
c = st.t.ppf(1 - (0.05/2), n1 + n2 -2)
c

1.9802722492407059

+c = 1.980 , -c = -1.980

The decision rule: if the t not equal to c, therefore reject Ho, otherwise accept Ho

b) Construct a 95% confidence interval for the difference between the mean contributions of
private business and government agency employees who contribute to United Way. Do the
hypothesis test and the confidence interval produce compatible results? Explain and give
reasons for your answer.

In [7]:
sp = m.sqrt(((n1 - 1)* m.pow(sd1,2) + (n2 - 1)* m.pow(sd2,2)) / (n1 + n2 -2))
sp

58.18807322170272

In [8]:
t = ((x1 - x2) - 0) / sp * m.sqrt(1/n1 + 1/n2)
t

-0.25139099417628497

t(-0.251,0.251) not equal to c(-1.980, 1.980), therefore Ho is rejected